apps/dashboard/src/components/workflow-editor/test-workflow/test-workflow-form.tsx

```diff
@@ -127,23 +127,23 @@ export const TestWorkflowForm = ({ workflow }: { workflow?: WorkflowResponseDto
             value={activeSnippetTab}
             onValueChange={(value) => setActiveSnippetTab(value as SnippetLanguage)}
           >
-            <TabsList className="border-t-0">
-              <TabsTrigger className={tabsTriggerClassName} value="framework">
+            <TabsList className="border-t-0" variant="regular">
+              <TabsTrigger className={tabsTriggerClassName} value="framework" variant="regular">
```

Comment:
How come the `regular` isn't the default? 🤔

Reply:
we have two variants now, and I wasn't sure which one should be a default... so decided to make the shadcn one a default